model-optimizer/unit_tests/mo/utils/cli_parser_test.py

```diff
@@ -908,21 +908,32 @@ def test_single_pass(self):
         self.assertEqual(parse_transform("LowLatency"), [("LowLatency", {})])
 
     def test_single_pass_with_args(self):
-        self.assertEqual(parse_transform("LowLatency[num_iterations=2]"),
-                         [("LowLatency", {"num_iterations": 2})])
+        self.assertEqual(parse_transform("LowLatency[use_const_initializer=True]"),
+                         [("LowLatency", {"use_const_initializer": True})])
+
+    def test_single_pass_with_args2(self):
+        self.assertEqual(parse_transform("LowLatency[sub_graph_iterations={'TI_name':2,'TI_name_2':-1}]"),
+                         [("LowLatency", {"sub_graph_iterations": {"TI_name": 2, "TI_name_2": -1}})])
```

Comment:
Could you please add negative tests cases with wrong dict attribute.

Reply:
how to distinguish wrong map from str?
`convert_string_to_real_type` function do not raise exceptions, just return `False`

Reply:
sub_graph_iterations argument has been removed